bundles/org.openhab.binding.pjlinkdevice/src/main/resources/OH-INF/i18n/pjLinkDevice.properties

```
@@ -16,42 +16,42 @@ thing-type.config.pjLinkDevice.pjLinkDevice.autoReconnectInterval.label = Auto R
 thing-type.config.pjLinkDevice.pjLinkDevice.ipAddress.label = IP Address
 thing-type.config.pjLinkDevice.pjLinkDevice.ipAddress.description = The address of the PJLink device to control.
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshInputChannel.label = Poll for Selected Input Channel
-thing-type.config.pjLinkDevice.pjLinkDevice.refreshInputChannel.description = Enable polling for the selected input channel. Only considered if the refreshInterval interval is greater than zero.
+thing-type.config.pjLinkDevice.pjLinkDevice.refreshInputChannel.description = Enable polling for the selected input channel. Only used if the refresh interval is greater than zero.
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshInterval.label = Refresh Interval (s)
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshInterval.description = How often to poll the device state (in seconds). A value of zero will disable polling.
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshLampState.label = Poll for Lamp State
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshLampState.description = Enable polling for the lamp state. Only considered if the refresh interval is greater than zero.
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshMute.label = Poll for Mute State
-thing-type.config.pjLinkDevice.pjLinkDevice.refreshMute.description = Enable polling for the mute state. Only considered if the refreshInterval interval is greater than zero.
+thing-type.config.pjLinkDevice.pjLinkDevice.refreshMute.description = Enable polling for the mute state. Only used if the refresh interval is greater than zero.
 thing-type.config.pjLinkDevice.pjLinkDevice.refreshPower.label = Poll for Power State
-thing-type.config.pjLinkDevice.pjLinkDevice.refreshPower.description = Enable polling for the power state. Only considered if the refreshInterval interval is greater than zero.
+thing-type.config.pjLinkDevice.pjLinkDevice.refreshPower.description = Enable polling for the power state. Only used if the refresh interval is greater than zero.
 thing-type.config.pjLinkDevice.pjLinkDevice.tcpPort.label = TCP Port
 thing-type.config.pjLinkDevice.pjLinkDevice.tcpPort.description = The TCP port of the PJLink device to control.
 
 # channel types
 
 channel-type.pjLinkDevice.audioAndVideoMute.label = Audio and Video Mute
-channel-type.pjLinkDevice.audioAndVideoMute.description = Select the audio and video mute status
+channel-type.pjLinkDevice.audioAndVideoMute.description = Set the audio and video mute state.
 channel-type.pjLinkDevice.audioMute.label = Audio Mute
-channel-type.pjLinkDevice.audioMute.description = Select the audio mute status
+channel-type.pjLinkDevice.audioMute.description = Set the audio mute state.
 channel-type.pjLinkDevice.input.label = Input
-channel-type.pjLinkDevice.input.description = Select the input signal to use
+channel-type.pjLinkDevice.input.description = Select the input source.
```

Should it be changed like the others?
```suggestion
channel-type.pjLinkDevice.input.description = Set the input source.
```